website/blog/2023-03-21-document-databases-definition-features-use-cases.md

```diff
@@ -0,0 +1,141 @@
+| Structure   | Flexible schema design, documents can have different structures within the same collection                             | Predefined, rigid structure with tables, columns, and rows                              |
+| Scalability | Efficient horizontal scaling, allowing for easy distribution of data across multiple hosts                             | Vertical scaling, adding resources to a single machine to handle more data              |
+| Querying    | Rich querying capabilities, with support for complex nested data structures                                            | Limited support for querying nested data structures, with more focus on join operations |
+| Development | Developer-friendly, with a more intuitive data model for object-oriented programming languages                         | Less intuitive data model, more complex SQL queries required                            |
+| Consistency | Lower consistency guarantees, with more focus on performance and availability                                          | Higher consistency guarantees, with more focus on data integrity and accuracy           |
+| Use Cases   | Best for handling large volumes of semi-structured or unstructured data, suited for modern web and mobile applications | Best for handling structured data, suited for traditional business applications         |
+
+**Read more:** PostgreSQL vs MongoDB - Understanding a Relational Database vs Document Database(To Link once published)
+
+## Examples of NoSQL Document Databases
+
+### FerretDB
+
+FerretDB is an open-source document database with MongoDB-compatibility built-in, allowing you to run MongoDB workloads on other database backends, such as PostgreSQL, Tigris, SAP Hana, and many others.
+
+### MongoDB
+
+MongoDB is a proprietary documented-oriented database built to be highly scalable and performant in handling high-volume data storage.
+Being one of the most popular document databases on the market, MongoDB has redefined the space with its range of drivers and APIs for various frameworks and programming languages.
+
+### Couchbase
+
+Couchbase is an open-source popular document database renowned for its scalability, performance, and speed.
+With a flexible model based on JSON documents, Couchbase enables effortless data modeling and schema changes.
+As a multi-model NoSQL database, Couchbase is tailored towards highly interactive applications requiring low latency and high throughput.
+
+### RavenDB
+
+RavenDB is a NoSQL document database with several features and capabilities suitable for mobile and web applications.
+Built with ACID compliance, RavenDB is fully transactional within the database and across clusters.
+RavenDB is an all-in-one database that includes a cloud service, a Time Series model, ML processing, and an OLAP plugin for business analysis.
```

Comment:
OLAP, I just googled to find out what it was :see_no_evil: maybe we don't need acronym.